src/frontend/components/info-panel/info-panel.ts

```diff
@@ -27,18 +27,20 @@ export class InfoPanel {
   constructor(
     private componentDataStore: ComponentDataStore,
     private userActions: UserActions,
+    private _ngZone: NgZone,
     @Inject(ElementRef) private elementRef: ElementRef
   ) {
 
     this.componentDataStore.dataStream
-      .filter((data: any) => data.action &&
-        data.action !== UserActionType.GET_DEPENDENCIES)
-      .debounce((x) => {
```

Comment:
No, we don't we fixed the batch call issue for which it was added in mutation observer one forgot to  remove this.